maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/MavenUtil.java

```diff
@@ -474,6 +474,47 @@ public static String getWarSourceDirectory( IMavenProjectFacade facade )
         return retval;
     }
 
+    public static boolean isMavenBundlePlugin( IProject project )
```

Comment:
THis method is not called from anywhere.  It should be called from the configurator.